IO-phone/src/app/poll/poll.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, inject } from '@angular/core';
+import { PollService } from './poll.service';
+
+@Component({
+  selector: 'app-poll',
+  standalone: true,
+  imports: [],
+  templateUrl: './poll.component.html',
+  styleUrl: './poll.component.css'
+})
+export class PollComponent implements OnInit {
+  private pollService = inject(PollService);
+
+  ngOnInit(): void {
+    //TODO: fetch a poll template from server and store it
```

Comment:
Minor: jeśli masz gdzieś na wierzchu ticket, który za to odpowiada, to możesz dodać `// TODO SCRUM-{numer}`. W tym przypadku jest to chyba `SCRUM-82`